src/status.rs

```diff
@@ -79,11 +79,11 @@ pub struct RadioShow {
 }
 
 /// The main Space API status object.
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, RustcDecodable)]
 pub struct Status {
 
     // Hackerspace properties
-    pub api: &'static str,
```

Comment:
Originally @rnestler and I decided to keep this as a static string because this is a static property - we can only implement versoin `0.13`, so the string doesn't need to be dynamically allocated.

@rnestler, do we want to change this to a String for consistency and flexibility for the future?